R/data-model-helpers.R

```diff
@@ -245,3 +245,50 @@ data_model_db_types_to_R_types <- function(data_model) {
   data_model$columns$type <- new_type
   data_model
 }
+
+upd_pks_after_rename <- function(
+  pks,
```

Comment:
This needs to come right after `function(` .

Reply:
This as in `pks,`?